policyengine-client/src/countries/uk/uk.jsx

```diff
@@ -17,32 +17,32 @@ const childNamer = {
 }
 
 function validatePolicy(policy, defaultPolicy) {
-    if(defaultPolicy) {
-        for(let parameter in policy) {
+    if (defaultPolicy) {
```

Comment:
Do you use a jsx formatter? This reformatting happened via my vscode's default.

Reply:
No, which VS Code one are you using? I'll use that